Core/Core/Inbox/AttachmentPicker/ViewModel/AttachmentPickerViewModel.swift

```diff
@@ -115,21 +141,41 @@ class AttachmentPickerViewModel: ObservableObject {
         interactor.addFile(url: url)
     }
 
+    func fileSelected(file: File) {
```

Comment:
Maybe this is only me, but I had to doublecheck what this method does because of the name.
IMHO in these cases `didSelectFile()` better conveys the message, that this is an action handler.
(or is this `didSomething` just  an outdated apple convention?)
Same for the method above with URL.